_posts/en/newsletters/2023-11-08-newsletter.md

```diff
@@ -0,0 +1,224 @@
+---
+title: 'Bitcoin Optech Newsletter #276'
+permalink: /en/newsletters/2023/11/08/
+name: 2023-11-08-newsletter
+slug: 2023-11-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Discontinuation of mailing lists:** administrators for the
```

Comment:
There will be a forthcoming announcement as to where the bitcoin-dev mailing list might be moving to; could this be reworded as something less than total death?

Reply:
Thank you, looks better now.